dll/win32/shell32/wine/shell32_main.c

```diff
@@ -961,6 +964,7 @@ VOID WINAPI Printers_UnregisterWindow(HANDLE hClassPidl, HWND hwnd)
 typedef struct
 {
     LPCWSTR  szApp;
+    LPCWSTR  szOSVersion;
```

Comment:
The location of this file implies that it's synced with Wine. Are we already diverging from upstream here, or should we consider ifdefs or whatever?

Reply:
I see Katayama has committed some changes to that file (as per the copyright header) however I'm not sure if he sent such changes to upstream first. For the sake of easier syncing, I'll just wrap ReactOS related coder in an ifdef.

Reply:
> dll/win32/shell32             # Forked at Wine-20071011

According to README.WINE